### PR TITLE
[5434] - new guidance for outside sign-off period

### PR DESCRIPTION
**Pull request by d-a-v-e**
### Context

Some of the guidance and reports content only makes sense inside the census or performance profiles sign off periods. For example, the performance profiles guidance mentions a deadline which is now in the past.

### Changes proposed in this pull request

* defines new copy in views
* adds `Setting` for `in_sign_off_period?` set to `false` for now

### CODE REVIEW

File: app/views/guidance/performance_profiles_outside.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      </ul>
+      <h2 class="govuk-heading-m" id="fixing-mistakes-in-your-trainee-data">Fixing mistakes in your trainee data</h2>
+      <p class="govuk-body">You’ll need to fix any mistakes before you sign off the data. How you’ll do this depends on whether you’re a provider of school centred initial teacher training (SCITT) or a higher education institution (HEI).</p>
+      <h2 class="govuk-heading-m" id="signing-off-your-trainee-data">Signing off your trainee data</h2>
+      <p class="govuk-body">A senior person from your organisation will need to complete a form to sign off the data. This should not be the same person who entered the data.</p>
+      <h2 class="govuk-heading-m" id="how-your-trainee-data-will-be-used">How your trainee data will be used</h2>
+      <p class="govuk-body">Read the ITT performance profiles methodology (opens in a new tab) to find out how your trainee data will be used.</p>
+      <p class="govuk-body">Some trainees will not be included in the ITT performance profiles publication. For example, trainees who withdrew within 90 days of the course start will not be included.</p>
+   </div>
+</div>
```
Comment (tavy88) [inline app/views/guidance/performance_profiles_outside.html.erb:32]: There's a few lines missing new line like here. I wonder why the linter doesn't capture these

File: config/settings.yml [hunk 2]
```diff
@@ -156,6 +156,7 @@ google:
 
 sign_off_trainee_data_url: https://example.com/trainee-data-sign-off
 sign_off_performance_profiles_url: https://forms.gle/AtAvteRTKJu3mbXS7
+in_sign_off_period?: false
```
Comment (tavy88) [inline config/settings.yml:159]: Do we need `?` here ?

File: app/controllers/guidance_controller.rb [hunk 3]
```diff
@@ -43,6 +47,14 @@ def current_academic_cycle
     @current_academic_cycle ||= AcademicCycle.current
   end
 
+  def signing_off_data
+    @signing_off_data ||= if Settings.in_sign_off_period?
+                            "Sign off your list of trainees from #{previous_academic_cycle.label} for the performance profiles publication"
+                          else
+                            "Signing off your list of trainees for the performance profiles publication"
+                          end
```
Comment (tavy88) [inline app/controllers/guidance_controller.rb:55]: Should these live in locales file ?

**Review by richardpattinson — APPROVED** [review]
Looks great, nice work! =)

**Comment by johndoates** [comment]
The content of the report and guidance pages looks good to me.

The title of the report needs to change to match the h1 heading. At the moment it's "Sign off your list of trainees from  for the performance profiles publication - Register trainee teachers - GOV.UK". 

It should be "Signing off your list of trainees for the performance profiles publication - Register trainee teachers - GOV.UK"

**Pull request merged**